concepts/ownership/about.md

```diff
@@ -1 +1,118 @@
 # Ownership
+In Cairo, the concept of ownership is attached to variables rather than the values they hold.
+This means that while multiple variables can refer to the same value, ownership is concerned with the variables themselves.
+Values in Cairo are immutable, meaning once a value is created, it cannot be changed.
+This immutability ensures that even if many variables refer to the same value, the value itself remains constant and unaltered.
+
+ ## The ownership rules.
+  _make sure to keep these rules in mind as you work with Cairo_
+```
+- Each variable in Cairo has an owner.
+- There can only be one owner at a time.
+- When the owner goes out of scope, the variable is destroyed.
+```
+### Variable scope
+Technically a scope is the range within a program in which a variable is valid
+
+```rust
+fn main(){
+     let x = 3;
+}
+   
+```
+the `x` is an integer which is valid at the point in which it is declared up to the end of the given scope, in this case `main()` function.
+
+```rust
+fn main(){ 
+                      //x is not valid here; it has not been declared
+     let x = 3;       //x is valid from this point forward ; it is been declared
+// do stuffs with x   // x is also valid here; it is still in scope
+
+} // this scope ended here and x goes out of scope and no longer valid.
+   
+```
+Note: 
```

Comment:
```suggestion

Note: 
```
nit: add spacing between separate components, paragraphs, examples etc., makes for a more nicely formatted file